middleware/requestid/requestid.go

```diff
@@ -49,3 +55,13 @@ func FromContext(c fiber.Ctx) string {
 	}
 	return ""
 }
+
+// FromUserContext returns the request ID from the UserContext.
+// If there is no request ID, an empty string is returned.
+// Compared to Local, UserContext is more suitable for transmitting requests between microservices
+func FromUserContext(ctx context.Context) string {
```

Comment:
This probably can be renamed `FromContext()`

Reply:
fix